src/main/java/atmosfears/AtmosFearsBE/service/CalculationService.java

```diff
@@ -0,0 +1,44 @@
+package atmosfears.AtmosFearsBE.service;
```

Comment:
Formatting in this file is a little more condensed than in other files. You used 2 spaces for indent in this file whereas we 4 spaces are used in the remaining files. Please reformat this file to be consistent.

You can also add some lines between parts of code inside the `getPredictionsForFile` method to make it easier to read.